stock_orderpoint_origin/models/procurement_group.py

```diff
@@ -15,9 +15,10 @@ def run(self, procurements, raise_user_error=True):
         Forecast = self.env["report.stock.report_product_product_replenishment"]
         new_procurements = []
         for procurement in procurements:
+            warehouse = procurement.values["warehouse_id"]
+            ForecastWH = Forecast.with_context(warehouse=warehouse.id)
```

Comment:
I think warehouse can be void too. That has to be taken into account.

Reply:
@rousseldenis Done